drafts/WEP0004-community-board.rst

```diff
@@ -0,0 +1,60 @@
+
+.. Template instructions:
+
+.. 0 Copy this file and rename it according to WEP filename convention: [WEP####-short-descriptive-name.rst]
+.. 1. Replace all values in square brackets [] with actual values.
+.. 2. Remove optional fields that you do not need. 
+.. 3. Follow reStructured Text markup conventions: http://docutils.sourceforge.net/rst.html
+.. 4. Remove these instructions before submitting the PR.
+.. 5. Submit the WEP as a PR and make sure it follows the guidelines of WEP0000: [URL]
+
+.. For questions and help in creating the WEP, email support@writethedocs.org
+
+:WEP-Number: 0004
+:Created: 2022-10-05
+:Last-Modified: [YYYY-MM-DD]
+:Author:  - Mikey Ariel
+          - Endorsed by CoreOps team
+:Status: Draft 
+:Replaces: WEP0003-wtd-org-structure-governance
+
+WEP0004 - Updated team structure and community board
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Abstract
+========
+
+This WEP further refines the WTD team structure to reflect a more accurate workload breakdown, renames the community council to community board with Mikey Ariel as the chair and coordinator, and clarifies more of the roles and responsibilities of each team and team leader in the organization. 
+
+To see the proposed changes, see this PR: https://github.com/writethedocs/www/pull/1811
+
+Motivation
+==========
+
+After we implemented the original team restructure WEP, we realized that the breakdown of teams was too granular compared to the actual workload distribution in the organization. 
+
+Link to the original team restructure WEP0003: https://github.com/writethedocs/weps/blob/main/accepted/WEP0003-wtd-org-structure-governance.rst
+
+We also recognized a gap in the community council, since the CoreOps team was entrenched in conference-driven work and we didn't have the capacity to launch the board meetings.
```

Comment:
```suggestion
We also recognized a gap in the community council, since the CoreOps team was entrenched in conference-driven work and didn't have the capacity to manage the community backlog and initiatives or launch the board meetings that we wanted to start holding.
```